src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/DefaultTranscriptsList.tsx

```diff
@@ -34,13 +37,19 @@ type Props = {
 const DefaultTranscriptslist = (props: Props) => {
   const { gene } = props;
   const sortedTranscripts = defaultSort(gene.transcripts);
+  const [isFilterOpen, setFilterOpen] = useState(false);
+
+  const toggleFilter = () => { isFilterOpen ? setFilterOpen(false) : setFilterOpen(true); }
 
   return (
     <div>
       <div className={styles.header}>
+        {isFilterOpen && <GeneFilter toggleFilter={toggleFilter} />}
         <div className={styles.row}>
-          <div className={styles.left}>Filter & sort</div>
-          <div className={styles.middle}></div>
+          <div className={isFilterOpen ?  styles.hidden : styles.filterLabel} onClick={toggleFilter}>
```

Comment:
Not that it particularly matters, but why use styles.hidden when you can just conditionally render or not render this element:

```
{ !isFilterOpen &&
  <div className={styles.filterLabel} onClick={toggleFilter}>
    Stuff
  </div>
}
```